.github/actions/prefix-pr/action.yml

```diff
@@ -0,0 +1,8 @@
+name: "Prefix PR"
```

Comment:
You also have to do something with the `create-jira-issues-for-community-activities` workflow because otherwise it'll create issues for every PR, see Teams.

Reply:
So if I understand that workflow correctly, it first checks if the title contains an issue key or not, and if it doesn't it creates a Jira issue. How about checking the branch instead of the title there, and determining if its an issue branch or not, instead of the PR title?

Reply:
Issue-specific PRs are possible to do from non-issue branches too (like https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/211) so it needs to keep checking the title.

Reply:
How about checking for both and going forward if one is true?

Reply:
That sounds good. DRY the branch-checking logic with the prefixing action; see the existing Scripts folder and how we load scripts from there.

Reply:
Between these two there is nothing to really DRY.